### DISCUSSION TITLE
Feature Request: Better Workflow view / more options to organize workflows

### DISCUSSION THREAD

**Discussion by SvenSchneiderDVAG** [community/community#45221] [Actions]
### Select Topic Area

Product Feedback

### Body

Hi all, I'd love to see an improvement for the Actions view, especially the left box where it's showing all the workflows in your repository. 

![image](https://user-images.githubusercontent.com/88534282/214309556-1b1ca6f1-76c2-4d82-b827-78cae85ab06f.png)

When you have more than 15+ workflows it's getting quite messy to find a workflow. The "Show more workflows ..." is simply not enough.

It could be improved by giving us folders to organize workflows or let us search or even filter our workflows. (newest/oldest/ascending/descending) etc. 

Is this already part of a road map? 

Kind regards,
Sven Schneider

**Comment by sbusch** [comment]
See also https://github.com/orgs/community/discussions/53383

**Comment by aura1213** [comment]
This is exactly what everybody want!
